devito/compiler.py

```diff
@@ -76,7 +76,7 @@ def __init__(self, *args, **kwargs):
         self.version = kwargs.get('version', None)
         self.cc = 'gcc' if self.version is None else 'gcc-%s' % self.version
         self.ld = 'gcc' if self.version is None else 'gcc-%s' % self.version
-        self.cflags = ['-O3', '-g', '-march=native', '-fPIC', '-Wall', '-std=c99',
+        self.cflags = ['-O3', '-g', '-fPIC', '-Wall', '-std=c99',
                        '-Wno-unused-result', '-Wno-unused-variable']
```

Comment:
This will go back to `native` I just leave it there for now to make my life easier